db/model/user.js

```diff
@@ -198,12 +195,7 @@ module.exports = function user(seq, dataTypes) {
             seq.models.Profile.findById(_this.profileId)
             .then((p) => {
               if (p) {
-                if (Number(p.isDeleted) === 0) {
-                  resolve();
-                } else {
-                  reject(new Error(
-                    `Profile ${_this.profileId} has been soft-deleted`));
-                }
+                resolve();
```

Comment:
ok to change this even though we're keeping paranoid for users?

Reply:
Not likely to be a problem for us, but you're right. I'll keep paranoid on for profiles, too, for now.